### PR TITLE
Update UG with command details and landing page with CI/codecov badges

### PR DESCRIPTION
**Pull request by Dinoman44**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -156,55 +156,58 @@ Examples:
...
 
 Format: `find [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]… [ps/PROJECT_STATUS] [py/PAYMENT_STATUS] [cs/CLIENT_STATUS] [d/DEADLINE]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-* Names only need to match the start of a word. e.g. `find n/Han` OR `find n/B`
-    matches `Hans Bo`
+* For finding by name, the search is case-insensitive. e.g `hans` will match `Hans`.
```
Comment (tanwartushar) [inline docs/UserGuide.md:165]: Not just name, find with any parameter is case-insensitive

File: docs/UserGuide.md [hunk 2]
```diff
@@ -156,55 +156,58 @@ Examples:
...
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
+* Persons matching at least one name keyword will be returned (i.e. `OR` search).
+  e.g. `find n/Han Bo` will return `Hans Gruber`, `Bo Yang`.
+* Names only need to match the start of a word. e.g. `find n/Han` OR `find n/B` matches `Hans Bo`.
 * Phone number, email, address, project status, payment status, client status must match the exact string
-  e.g. `cs/in progress` will not match `cs/in prog`
-* Only 1 tag needs to be matched for a person to be found
+  e.g. `cs/in progress` will not match `cs/in prog`.
```
Comment (tanwartushar) [inline docs/UserGuide.md:171]: "in progress" isn't a client status, use ps/

**Review by tanwartushar — COMMENTED** [review]
minor issues, merge once fixed 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (Dinoman44) [reply docs/UserGuide.md:165]: I just realized there is already a line that states this; will just move it here.

**Pull request merged**